docs/setup_docs.sh

```diff
@@ -36,24 +36,33 @@ function integrate_connector_docs {
 }
 
 
+SKIP_INTEGRATE_CONNECTOR_DOCS=false
+for arg in "$@"; do
+  if [ "$arg" == "--skip-integrate-connector-docs" ]; then
+    SKIP_INTEGRATE_CONNECTOR_DOCS=true
+    break
+  fi
+done
+
 # Integrate the connector documentation
+if [ "$SKIP_INTEGRATE_CONNECTOR_DOCS" = false ]; then
```

Comment:
I think we need to check whether the directory exists or not as well.

I `rm -rf` the connectors directory and try `./build_docs.sh --skip-integrate-connector-docs`. Unfortunately, the error is thrown.
![image](https://github.com/apache/flink/assets/55568005/820a4c0e-87c7-4568-b97d-bd47839e6960)

Reply:
done!